docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,162 @@
+.. _acsincome-data:
+ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset [2]_.
+Briefly, the UCI Adult dataset is commonly used as a benchmark dataset when comparing
```

Comment:
I'm not sure what "Briefly" is meant to indicate here tbh. Perhaps it's because English is not my native language (?)